### ISSUE TITLE
Accessibility: Cannot select the elements on focus in PDP

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#6574]
**Environment Details**
Spartacus: epic/accessibility branch
Browser: Chrome & firefox & Safari
Device: mac
Happens on: http://dev-com-3.accdemo.b2c.ydev.hybris.com:4200/electronics-spa/en/USD

**Steps to Reproduce**
- Go to Product Details Page

We cannot actually select these by pressing ENTER or SPACEBAR key:
- Show reviews
<img width="1048" alt="Screen Shot 2020-02-14 at 1 13 00 PM" src="https://user-images.githubusercontent.com/45409515/74557118-80212580-4f2d-11ea-904f-2484c40f92e6.png">

- Product tabs
<img width="1119" alt="Screen Shot 2020-02-14 at 1 14 08 PM" src="https://user-images.githubusercontent.com/45409515/74557163-9929d680-4f2d-11ea-9548-c5c67ada0549.png">

- Review stars
<img width="965" alt="Screen Shot 2020-02-14 at 1 15 26 PM" src="https://user-images.githubusercontent.com/45409515/74557193-ac3ca680-4f2d-11ea-82c4-c9bfe0bb9185.png">

**Comment by psmul** [comment]
QA steps:
- go to any PDP
- tab until "Show reviews" label on top
- space or enter element

Expected result:
- should scroll down to tab with reviews

---------------------

QA steps:
- start new form with review for product


Expected result:
- should be able to tab through form and select star rating with SPACE

**Comment by mszczepaniak** [comment]
Tested successfully.
